packages/trader/src/AppV2/Containers/Trade/__tests__/trade.spec.tsx

```diff
@@ -144,6 +144,14 @@ describe('Trade', () => {
         expect(screen.getByTestId('dt_trade_loader')).toBeInTheDocument();
     });
 
+    it('should show loader if we are switching from demo to real account', () => {
+        default_mock_store.client.is_switching = true;
+        default_mock_store = mockStore({});
```

Comment:
But you are overwriting the default_mock_store in line 149, what is the point then in changing value in 148 line?